.editorconfig

```diff
@@ -22,14 +22,12 @@ insert_final_newline = true
 insert_final_newline = false
 charset = utf-8-bom
 
-[project.json]
-insert_final_newline = false
-charset = utf-8-bom
-
-[*.nuget.targets]
-insert_final_newline = false
-charset = utf-8-bom
-
 [*.json]
 # 2-column space indentation
 indent_size = 2
+
+[*.cs]
+dotnet_style_qualification_for_event = true:silent
+dotnet_style_qualification_for_field = true:silent
+dotnet_style_qualification_for_method = true:silent
+dotnet_style_qualification_for_property = true:silent

```

Comment:
VS was warning me for every `this.` prefix and suggesting to remove them, so I thought we might as well put the rule explicitly here (it's currently enforced by StyleCop). We could also add some naming rules here as well, I guess.